subcommand/consul-sidecar/command.go

```diff
@@ -44,6 +56,10 @@ func (c *Command) init() {
 		"Log verbosity level. Supported values (in order of detail) are \"trace\", "+
 			"\"debug\", \"info\", \"warn\", and \"error\". Defaults to info.")
 
+	c.flagSet.BoolVar(&c.flagEnableMetricsMerging, "enable-metrics-merging", false, "Enables consul sidecar to run a merged metrics endpoint. Defaults to false.")
+	c.flagSet.StringVar(&c.flagMergedMetricsPort, "merged-metrics-port", "20100", "Port to serve merged Envoy and application metrics. Defaults to 20100.")
+	c.flagSet.StringVar(&c.flagServiceMetricsPort, "service-metrics-port", "0", "Port where application metrics are being served. Defaults to 20100.")
```

Comment:
You've got the default value set to "0" and the text says "defaults to 20100", assuming this is a typo?
```suggestion
	c.flagSet.StringVar(&c.flagServiceMetricsPort, "service-metrics-port", "20100", "Port where application metrics are being served. Defaults to 20100.")
```

Reply:
So this one shouldn't default to 20100. I had made a typo in the Defaults sentence. It is only set in connect-inject/consul_sidecar.go if: metrics and metrics merging are enabled, and the service metrics port is greater than 0. So I thought I could default `serviceMetricsPort` to 0, since if the merged metrics server is not running, it will not get used, and if it is running, it will definitely get set to something that is not 0. I changed the sentence to "Defaults to 0." I added a comment to explain this as well, let me know if its clear.